docs/guide/sessions.md

```diff
@@ -52,45 +52,62 @@ Symfony can be configured to use custom handlers, or just fall back to what is
-        # handler_id set to null will use default session handler from php.ini
-        handler_id:  ~
+        # handler_id can be set to null (~) like default in Symfony, if so will use default session handler from php.ini
+        # But in order to use %ezplatform.session.save_path%, default eZ Platform instead sets %ezplatform.session.handler_id% to:
+        # - session.handler.native_file (default)
+        # - ezplatform.core.session.handler.native_redis (recommended value for Cluster usage, using php-redis session handler )
+        handler_id: '%ezplatform.session.handler_id%'
 ```
 
 ### Recommendations for production setup
 
 #### Single-server setup
 
-For a single server, the default handler is preferred.
+For a single server, the default file handler is preferred.
 
 #### Cluster setup
 
-For a [cluster](clustering.md) setup you need to configure sessions to use a back end that is shared between web servers and supports locking.
-The only options out of the box supporting this in Symfony are the native PHP Memcached session save handler
-provided by the `php-memcached` extension, and Symfony session handler for PDO (database).
+For a [cluster](clustering.md) setup you need to configure sessions to use a backend that is shared between web servers.
+The options out of the box in Symfony are the native PHP Memcached or Redis session handlers, and Symfony session handler for PDO _(database)_.
+
+To avoid concurrent access to session data from frontedn nodes, if possible you should either:
+- Enable [Session locking](http://php.net/manual/en/features.session.security.management.php#features.session.security.management.session-locking)
+- Use "Sticky Session", aka [Load Balancer Persistence](https://en.wikipedia.org/wiki/Load_balancing_(computing)#Persistence) 
+
+Session locking is avaiable with `php-memcached`, and with `php-redis` _(v4.2.0 and higher)_.
 
 ##### Storing sessions in Memcached using `php-memcached`
 
-To set up eZ Platform using Memcached you need to [configure the session save handler settings in `php.ini`](http://php.net/manual/en/memcached.sessions.php),
-and optionally tweak [`php-memcached` session settings](http://fr2.php.net/manual/en/memcached.configuration.php).
+To set up eZ Platform using [Memcached](https://pecl.php.net/package/memcached) you need to [configure the session save handler settings in `php.ini`](http://php.net/manual/en/memcached.sessions.php), optionally tweak [`php-memcached` session settings](http://php.net/manual/en/memcached.configuration.php), and set eZ Platform's `%ezplatform.session.handler_id%`
+to `~` _(null)_.
+
+NOTE: For `php-memcached`, session locking should be kept enabled.
+
+##### Storing sessions in Redis using `php-redis`
 
-##### Storing sessions in Redis using pecl package
+To set up eZ Platform using the [Redis](https://pecl.php.net/package/redis)
+you need to set eZ Platform's `%ezplatform.session.handler_id%` to `ezplatform.core.session.handler.native_redis`,
```

Comment:
So, this is not completely correct. Unless you are on Platform.sh you also may configure redis using php.ini settings.
On platform.sh however, you must use `ezplatform.core.session.handler.native_redis` ( as you don't know the redis configuration at commit/build time )